gusto/expressions.py

```diff
@@ -0,0 +1,202 @@
+"""
+Some thermodynamic expressions to help declutter the code.
+"""
+from firedrake import exp
```

Comment:
Why not call this file `thermodynamics`, and remove all the `_expr` from the function names.

Then in the gusto code, rather than importing all of these into the global namespace, you can do:

```python
from gusto import thermodynamics

...

Pi = thermodynamics.pi(...)
theta = thermodynamics.theta(...)
```
etc...

Reply:
Sure, thank you!